aggregators/1.1/modules/ROOT/pages/aggregators-module-reference.adoc

```diff
@@ -205,7 +205,7 @@ Lastly, when elements that reach group-based aggregators come from a sequence th
 | Group Size | Number | The maximum size to assign to the group with the group ID resolved. All messages with the same group ID must have the same group size. If not, only the first resolved group size is considered correct. A warning is logged for every group that does not match. | #[itemSequenceInfo.sequenceSize] |
 | Eviction Time | Number | The time to remember a group ID after it is completed or timed out (0 means: don't remember, -1: remember forever) | 180 |
 | Eviction Time Unit | Time Unit | The time unit for the Eviction Time. | SECONDS |
-| Timeout | Number |  The maximum time to wait for the aggregation of a group to complete. If the timeout is reached before the total number of elements in that group is equal to the group's size, the aggregation is considered complete. To avoid constant group timeouts, a value of `0` is not supported. | -1(UNLIMITED) |
+| Timeout | Number |  The maximum time to wait for the aggregation of a group to complete. If the timeout is reached before the total number of elements in the group equals the group’s size, the aggregator triggers `notifyListenerOnTimeout`. This allows timed-out aggregations to be picked up by the Aggregator Listener if `IncludeTimedOutGroups` is set to `true`, without marking them as complete. Additionally, the `IncludeTimedOutGroups` parameter determines whether these timed-out groups are passed to the listener without marking the aggregation as complete.. To avoid constant group timeouts, a value of `0` is not supported. | -1(UNLIMITED) |
```

Comment:
```suggestion
| Timeout | Number |  The maximum time to wait for the aggregation of a group to complete. If the timeout is reached before the total number of elements in the group equals the group’s size, the aggregator triggers `notifyListenerOnTimeout`. This allows timed-out aggregations to be picked up by the Aggregator Listener if `IncludeTimedOutGroups` is set to `true`, without marking them as complete. Additionally, the `IncludeTimedOutGroups` parameter determines whether these timed-out groups are passed to the listener without marking the aggregation as complete. To avoid constant group timeouts, a value of `0` is not supported. | -1(UNLIMITED) |
```